src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -171,7 +171,8 @@ module Mutations = struct
       ~typ:(non_null bool)
       ~args:Arg.[ arg "filter" ~typ:(non_null (list (non_null string))) ]
       ~resolve:(fun { ctx = t; _ } () filter ->
-        Result.is_ok @@ Mina_lib.start_filtered_log t filter )
+        Result.is_ok
+        @@ Mina_lib.start_filtered_log ~commit_id:"not specified" t filter )
```

Comment:
Surely `commit_id` should be optional if you don't have it here. That said, it would make more sense to store it in `Mina_lib.t` and pass it through

Reply:
I made the `commit_id` to be correctly passed here instead of "not specified" in a follow-up commit